### PR TITLE
add more contact persons

### PR DESCRIPTION
**Pull request by florianthiery**
add more contact persons

### CODE REVIEW

**Comment by florianthiery** [comment]
> Greay, thanks =)
> 
> You know that you can also add other links if you want @florianthiery, or logos, etc. this would then appear in the upper left corner for the Research Squirrell Engineers
> 
> image
> 
> see [the docs](https://mmistakes.github.io/minimal-mistakes/docs/authors/) for some examples.

coll, I will try it

**Comment by florianthiery** [comment]
does it matter in which of the "persons" I put the info? the first one? @Chilipp

**Comment by Chilipp** [comment]
> the first one?

yes, it needs to be the first one @florianthiery, because only one author is displayed here by default (see https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/author-profile.html). Actually, I saw that you can also make it easier than using this *links* kind-of-thing. You can also simply add an email attribute. In other words:

```yaml
authors:
  - email: rse@fthiery.de
    name: Florian Thiery / Research Squirrel Engineers
  - email: rse@fthiery.de
    name: second author
```
works as well

**Comment by florianthiery** [comment]
I added some info, could you pls check them :-) @Chilipp

File: _bazaar/rse-in-the-digital-humanties-domain.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+      url: mailto:hi@squirrel.link
+    - icon: fas fa-fw fa-link
+      label: Website
+      url: http://squirrel.link
+    - icon: fas fa-fw fa-twitter-square
+      label: Twitter
+      url: https://twitter.com/RSESquirrel
+    - icon: fas fa-fw fa-github-square
+      label: Github
+      url: https://github.com/research-Squirrel-Engineers
```
Comment (Chilipp) [inline _bazaar/rse-in-the-digital-humanties-domain.md:15]:
```suggestion
  email: hi@squirrel.link
  uri: http://squirrel.link
  twitter: RSESquirrel
  github: research-Squirrel-Engineers
```
it's not documented, but these are all implemented in the [author-profile](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/author-profile.html)

**Comment by Chilipp** [comment]
alright, think about https://github.com/RSE-leaders/SORSE20/pull/120#discussion_r443575802 and commit the suggestion, if you like. Afterwards you are good to go and merge this PR

**Comment by florianthiery** [comment]
ok lets give it a try ;-)

**Pull request merged**